projects/components/src/header/page/page-header.component.ts

```diff
@@ -29,7 +29,13 @@ import { NavigableTab } from '../../tabs/navigable/navigable-tab';
           ></ht-icon>
 
           <ht-label [label]="titlecrumb.label"></ht-label>
-          <ht-beta-tag class="beta" *ngIf="this.isBeta"></ht-beta-tag>
+          <ht-label-tag
+            *ngIf="this.isBeta"
+            class="beta"
+            label="Beta"
```

Comment:
It seems like there's still an argument for a beta tag component even if it's noow built on the lower level label-tag component - better a tiny component than having to pass the same 3 params in every place we need a beta tag and updating them all together.

Reply:
That makes sense. Will add a `beta` tag component.